strongbox-parent/pom.xml

```diff
@@ -51,6 +51,8 @@
         <!-- Strongbox properties: -->
         <strongbox.home>${project.build.directory}/strongbox</strongbox.home>
         <strongbox.vault>${project.build.directory}/strongbox-vault</strongbox.vault>
+        <strongbox.tmp>${project.build.directory}/strongbox/tmp</strongbox.tmp>
```

Comment:
I don't think we need to be defining these at all -- they should work out of the box.